kubernetes/test-infra/minikube-on-packet/20-install-minikube.sh

```diff
@@ -1,10 +1,15 @@
+#!/usr/bin/bash
+
 # Install kubectl
+
 KUBECTL_VERSION=$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)
 curl -LO https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl \
-  && chmod +x kubectl && mv kubectl /usr/local/bin/
+  && chmod a+x kubectl && mv kubectl /usr/local/bin/
 kubectl version
 
 # Install minikube
 curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 \
-  && chmod +x minikube && mv minikube /usr/local/bin/
+  && chmod a+x minikube && mv minikube /usr/local/bin/
 minikube version
+minikube config set WantReportErrorPrompt false
```

Comment:
Nitpick: Can you explain what this does in a brief comment above the line, please?

Reply:
If minikube fails for any reason and this is set to `true` (the default) then it will show an interactive prompt asking the user to confirm that minikube can phone back home with diagnostic information. It's undesirable to have this happen in an automated environment like CI. 

The actual prompt message is this
```
================================================================================
An error has occurred. Would you like to opt in to sending anonymized crash
information to minikube to help prevent future errors?
To opt out of these messages, run the command:
	minikube config set WantReportErrorPrompt false
================================================================================
```